<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbuniq</class>
 <widget class="QDialog" name="dbuniq">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>268</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>349</width>
    <height>268</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>349</width>
    <height>268</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Уникальность текста</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/main/res/uniqueness.png</normaloff>:/main/res/uniqueness.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QSpinBox" name="spin_length">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>48</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>18</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Длина шингла</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя таблицы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_tablename">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>40</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_field">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>90</y>
     <width>325</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Искать текст в бд:</string>
   </property>
   <widget class="QRadioButton" name="w_radi_number">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>По номеру поля</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="w_radio_name">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>По именю поля</string>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="spin_field">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>48</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_exit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_do">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Проверить</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="w_progress">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>230</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
